src/Alb/Alb.Autorest/docs/New-AzAlb.md

```diff
@@ -144,6 +146,96 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SystemDataCreatedAt
```

Comment:
These are something that should be not generated. I use the same README.md as you, but those parameters are not generated. Would you please do following things.

1. Sync your code with latest main
2. run 'autorest --reset' to upgrade autorest.powershell
3. Regenerate the code with autorest.powershell